From 7a70094824d8337cf928002ad56116a5e1b864cc Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:04:18 -0400
Subject: [powerpc] perf: correct typos in counter enumeration

Message-id: <1410545655-205645-30-git-send-email-dzickus@redhat.com>
Patchwork-id: 94229
O-Subject: [RHEL7 PATCH 029/626] powerpc/perf: correct typos in counter enumeration
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit a9a5cda06922545ed7540cf5ed7e5ad6b80851d9
Author: Lijun Pan <LIJUN.PAN@FREESCALE.COM>
Date:   Wed Jun 5 15:01:32 2013 -0500

    powerpc/perf: correct typos in counter enumeration

    Signed-off-by: Lijun Pan <Lijun.Pan@freescale.com>
    Signed-off-by: Scott Wood <scottwood@freescale.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/include/asm/reg_fsl_emb.h b/arch/powerpc/include/asm/reg_fsl_emb.h
index 77bb71c..1cf8ab0 100644
--- a/arch/powerpc/include/asm/reg_fsl_emb.h
+++ b/arch/powerpc/include/asm/reg_fsl_emb.h
@@ -17,8 +17,8 @@
 /* Freescale Book E Performance Monitor APU Registers */
 #define PMRN_PMC0 0x010 /* Performance Monitor Counter 0 */
 #define PMRN_PMC1 0x011 /* Performance Monitor Counter 1 */
-#define PMRN_PMC2 0x012 /* Performance Monitor Counter 1 */
-#define PMRN_PMC3 0x013 /* Performance Monitor Counter 1 */
+#define PMRN_PMC2 0x012 /* Performance Monitor Counter 2 */
+#define PMRN_PMC3 0x013 /* Performance Monitor Counter 3 */
 #define PMRN_PMLCA0 0x090 /* PM Local Control A0 */
 #define PMRN_PMLCA1 0x091 /* PM Local Control A1 */
 #define PMRN_PMLCA2 0x092 /* PM Local Control A2 */
@@ -55,8 +55,8 @@
 
 #define PMRN_UPMC0 0x000 /* User Performance Monitor Counter 0 */
 #define PMRN_UPMC1 0x001 /* User Performance Monitor Counter 1 */
-#define PMRN_UPMC2 0x002 /* User Performance Monitor Counter 1 */
-#define PMRN_UPMC3 0x003 /* User Performance Monitor Counter 1 */
+#define PMRN_UPMC2 0x002 /* User Performance Monitor Counter 2 */
+#define PMRN_UPMC3 0x003 /* User Performance Monitor Counter 3 */
 #define PMRN_UPMLCA0 0x080 /* User PM Local Control A0 */
 #define PMRN_UPMLCA1 0x081 /* User PM Local Control A1 */
 #define PMRN_UPMLCA2 0x082 /* User PM Local Control A2 */
-- 
1.7.1